### PR TITLE
Day three

### PR DESCRIPTION
**Pull request by Munsa1**
In the following pull request, we have done the following 
- Added styling to our Awesome book site
- used bootstrap framework to implement styling
- checked linter errors
😎 Look forward to your feedback

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hello @Munsa1,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
 - The Project is code clean, however still some minor issues check my comment down below 👇

## Optional suggestions

- You can use  bootstrap for the output list [ Bootstrap forms](https://getbootstrap.com/docs/4.0/components/forms/) 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Assets/style.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+li {
+  list-style: none;
+  padding: 6px;
+  font-size: 26px;
+  font-weight: 500;
+  background-color: rgb(118, 163, 231);
+  display: grid;
+  grid-template-columns: 70% 30%;
```
Comment (dasileker) [inline Assets/style.css]: - [ ] Please add margin to separate between the lists.
- I can see that space is really tight, and the title and the author also.

Comment (Munsa1) [reply Assets/style.css]: thank you for the feedback it's done. please have a look

**Review by ZawadiSumba66 — APPROVED** [review]
#### Hi @Munsa1 && @sheylaPozo 👋 , 

### STATUS: APPROVED ✔️ ✔️ 
Great job implementing the change requested by the previous reviewer. 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**